pkg/security/registry/podsecuritypolicyreview/rest_test.go

```diff
@@ -0,0 +1,96 @@
+package podsecuritypolicyreview
+
+import (
+	"reflect"
+	"testing"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/client/cache"
+	clientsetfake "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/fake"
+
+	oscache "github.com/openshift/origin/pkg/client/cache"
+	admissionttesting "github.com/openshift/origin/pkg/security/admission/testing"
+	securityapi "github.com/openshift/origin/pkg/security/api"
+	oscc "github.com/openshift/origin/pkg/security/scc"
+)
+
+func TestNoErrors(t *testing.T) {
```

Comment:
The test cases are not covering error paths at all, see for yourself running:
`COVERAGE_OUTPUT_DIR=/tmp/origin_coverage/ hack/test-go.sh pkg/security/registry/podsecuritypolicyreview/`